src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,21 @@
+package duke.command;
+
+import duke.task.TaskList;
+
+/**
+ * Abstract class command.
+ */
+public abstract class Command {
+
+    /**
+     * To execute the command.
+     *
+     * @param tasks The tasks to be executed.
+     */
+    public abstract void execute(TaskList tasks);
+
+    /**
+     * @inheritDoc
+     */
+    public abstract boolean isExit();
```

Comment:
Is there a reason why you made this an abstract method instead of concretizing it and returning `false` most of the time?